base/sparse/cholmod.jl

```diff
@@ -125,27 +133,28 @@ function __init__()
                  configuration or by downloading the OS X or generic
                  Linux binary from www.julialang.org, which include
                  the correct versions of all dependencies.
-             """)
+            """)
         end
 
         ### Initiate CHOLMOD
         ### The common struct. Controls the type of factorization and keeps pointers
         ### to temporary memory.
-        global const commonStruct = fill(0xff, common_size)
+        resize!(commonStruct, common_size)
+        fill!(commonStruct, 0xff)
 
-        global const common_supernodal =
+        common_supernodal[] =
```

Comment:
You can probably remove the `convert`.

Reply:
Yeah. 
Thank you for the review and suggested improvements!